src/MauiReactor.TemplatePack/templates/items/MauiReactor.Component/.template.config/template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Adolfo Marinucci - James Alickolli",
+  "classifications": [ "Maui", "Code" ],
+  "identity": "MauiReactor.Component",
+  "name": ".NET MAUI Reactor Component",
+  "description": "A Maui reactor based component",
+  "shortName": "maui-reactor-component",
+  "sourceName": "MauiReactor.Component",
```

Comment:
the sourceName shouldn't be MauiReactorComponent?

Reply:
Just pushed a commit that resolves this